R/plotDispersion.R

```diff
@@ -5,7 +5,7 @@
 #' the plot can instead be Biological Coefficient of Variation (BCV is the square root of
 #' dispersion) against AveLogCPM.
 #'
-#' @param dispersionObj Counts matrix or DGEList (Required)
+#' @param DGEdata Counts matrix or DGEList (Required)
```

Comment:
@kar-agg-gen This feels like a great change - I made up `dispersionObj` and really hated it, so `DGEdata` makes a lot more sense.